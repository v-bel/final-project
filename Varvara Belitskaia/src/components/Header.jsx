/* eslint-disable import/prefer-default-export */
import React from 'react';
import { NavLink } from 'react-router-dom';

export const Header = () => {
  return (
    <nav className="navbar bg-light nav-pills justify-content-start mb-2">
      <NavLink
        exact
        to="/"
        className="nav-link"
        activeClassName="nav-link active"
      >
        Pokemons
      </NavLink>
      <NavLink
        to="/caught"
        className="nav-link"
        activeClassName="nav-link active"
      >
        My Pokemons
      </NavLink>
    </nav>
  );
};
