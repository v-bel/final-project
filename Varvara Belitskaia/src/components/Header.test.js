import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Header } from './Header';

it('should render the header with navigation links', () => {
  const enzymeWrapper = shallow(<Header />);
  expect(toJson(enzymeWrapper)).toMatchSnapshot();
});
