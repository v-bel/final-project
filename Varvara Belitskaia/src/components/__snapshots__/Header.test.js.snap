// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the header with navigation links 1`] = `
<nav
  className="navbar bg-light nav-pills justify-content-start mb-2"
>
  <NavLink
    activeClassName="nav-link active"
    aria-current="page"
    className="nav-link"
    exact={true}
    to="/"
  >
    Pokemons
  </NavLink>
  <NavLink
    activeClassName="nav-link active"
    aria-current="page"
    className="nav-link"
    to="/caught"
  >
    My Pokemons
  </NavLink>
</nav>
`;
